marimo/_cli/cli.py

```diff
@@ -451,6 +451,14 @@ def _get_stdin_contents() -> str | None:
     type=click.Path(),
 )
 @click.argument("args", nargs=-1, type=click.UNPROCESSED)
+@click.option(
+    "--timeout",
+    required=False,
+    default=None,
+    show_default=False,
+    type=float,
+    help="Enable a global timeout to shut down the server after specified number of minutes of no connection",
+)
```

Comment:
Im not sure if its required, but can you move this before the positional `args` and `name`